### ISSUE TITLE
imputation data analysis

### ISSUE THREAD

**Issue by yunaibiao** [jiabowang/GAPIT#113]
hello, GAPIT can be used for imputing missing data.

**Comment by jiabowang** [comment]
No, in the GAPIT the missing values just were imputed as heterozygotes.

**Comment by yunaibiao** [comment]
[GAPIT.Association.Manhattan_Geno.Backfat_of_waist.pdf](https://github.com/jiabowang/GAPIT/files/14837393/GAPIT.Association.Manhattan_Geno.Backfat_of_waist.pdf)
Hello,
Thank you for reaching out regarding your GWAS analysis using BLINK on imputated genotypes. It seems like you're encountering a discrepancy between the expected and observed results, with numerous points surpassing the threshold line, which differs significantly from your anticipated outcome. Typically, with chip results, you'd expect only a few scattered points, yet with the imputed genotypes, there are many points exceeding the threshold with relatively high p-values. This has raised concerns and confusion about the validity of the results. why?

**Comment by jiabowang** [comment]
@yunaibiao 
seems like the population structure was not control  enough. Or some other major factor was not considered in the model.
